crm/modules/member/command.inc.php

```diff
@@ -308,7 +308,8 @@ function command_member_membership_delete () {
     }
     // Delete membership
     member_membership_delete($esc_post['sid']);
-    return crm_url("contact&cid=$_POST[cid]&tab=plan");
+    message_register("Deleted membership");
+    return crm_url("contact&cid=${_POST['cid']}&tab=plan");
```

Comment:
adding the single quote marks around cid causes a PHP fatal error, think this will need tobe added in & the string concatenated in a different way

Reply:
Ah, missed that! if that's the case it'll need updated in quite a few places